openpype/hosts/maya/plugins/create/create_rig.py

```diff
@@ -20,6 +20,13 @@ def create(self, subset_name, instance_data, pre_create_data):
         instance_node = instance.get("instance_node")
 
         self.log.info("Creating Rig instance set up ...")
+        # TODO：change name (_controls_set -> _rigs_SET)
         controls = cmds.sets(name=subset_name + "_controls_SET", empty=True)
+        # TODO：change name (_out_SET -> _geo_SET)
```

Comment:
Same about this name change. The `_out_SET` doesn't necessarily ONLY include geo. It could also have locators that are used in a next step for specific lighting usages.